kubernetes-maven-plugin/plugin/src/test/java/org/eclipse/jkube/maven/plugin/mojo/build/BuildMojoTest.java

```diff
@@ -52,7 +53,7 @@ void buildServiceConfigBuilder_shouldReturnNonNullResourceConfigIfConfigured() {
         BuildServiceConfig.BuildServiceConfigBuilder buildServiceConfigBuilder = buildMojo.buildServiceConfigBuilder();
         // Then
         assertThat(buildServiceConfigBuilder.build()).isNotNull()
-            .returns("src/main/jkube", c -> c.getResourceDir().getPath())
+            .returns(new File("src/main/jkube"), c -> c.getResourceDir())
```

Comment:
Maybe you can use lambda here

Reply:
I'm not sure how would we do that? Is there a way to introduce a lambda expression here?

Reply:
Oh okay. I'll send in a commit.
Thanks.